core/src/test/java/org/apache/spark/unsafe/map/AbstractBytesToBytesMapSuite.java

```diff
@@ -217,19 +217,6 @@ public void setAndRetrieveAKey() {
       Assert.assertArrayEquals(valueData,
         getByteArray(loc.getValueBase(), loc.getValueOffset(), recordLengthBytes));
 
-      try {
```

Comment:
This test has the same problem as the above, but, is actually further just 'wrong' since a long time ago, when putNewValue was replaced by append. It is no longer expected to assert, or that's what I work out from the commit log.